src/main/java/com/gamingmesh/jobs/container/Job.java

```diff
@@ -227,15 +220,15 @@ public JobInfo getJobInfo(ActionInfo action, int level) {
      * @return the job name
      */
     public String getName() {
-	return jobName;
+	return fullName;
     }
 
     /**
-     * Get the job full name
-     * @return the job full name
+     * Get the job name in the config
+     * @return the job name in the config

```

Comment:
"in the config" should be `from the config`

Reply:
Updated.